src/axom/mir/examples/concentric_circles/mir_concentric_circles.cpp

```diff
@@ -0,0 +1,196 @@
+namespace mir = axom::mir;
+namespace bputils = axom::mir::utilities::blueprint;
+
+using RuntimePolicy = axom::runtime_policy::Policy;
+
+//--------------------------------------------------------------------------------
+void conduit_debug_err_handler(const std::string &s1, const std::string &s2, int i1)
+{
+  SLIC_ERROR(
+    axom::fmt::format("Error from Conduit: s1={}, s2={}, i1={}", s1, s2, i1));
+  // This is on purpose.
+  while(1)
+    ;
+}
+
+//--------------------------------------------------------------------------------
+void printNode(const conduit::Node &n)
+{
+  conduit::Node options;
+  options["num_children_threshold"] = 10000;
+  options["num_elements_threshold"] = 10000;
+  n.to_summary_string_stream(std::cout, options);
+}
+
+//--------------------------------------------------------------------------------
+int runMIR(RuntimePolicy policy,
+           int gridSize,
+           int numCircles,
+           const std::string &outputFilePath)
+{
+  // Initialize a mesh for testing MIR
+  auto timer = axom::utilities::Timer(true);
+  mir::MeshTester tester;
+  conduit::Node mesh;
+  {
+    AXOM_ANNOTATE_SCOPE("generate");
+    tester.initTestCaseFive(gridSize, numCircles, mesh);
+    // printNode(mesh);
+  }
+  timer.stop();
+  SLIC_INFO("Mesh init time: " << timer.elapsedTimeInMilliSec() << " ms.");
+
+  // Output initial mesh.
+  {
+    AXOM_ANNOTATE_SCOPE("save_input");
+    conduit::relay::io::blueprint::save_mesh(mesh, "concentric_circles", "hdf5");
+  }
+
+  // Begin material interface reconstruction
+  timer.start();
+  conduit::Node options, resultMesh;
+  options["matset"] = "mat";
+
+  int retval = 0;
+  if(policy == RuntimePolicy::seq)
+  {
+    retval = runMIR_seq(mesh, options, resultMesh);
+  }
+#if defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE)
+  #if defined(AXOM_USE_OPENMP)
+  else if(policy == RuntimePolicy::omp)
+  {
+    retval = runMIR_omp(mesh, options, resultMesh);
+  }
+  #endif
+  #if defined(AXOM_USE_CUDA)
+  else if(policy == RuntimePolicy::cuda)
+  {
+    constexpr int CUDA_BLOCK_SIZE = 256;
+    using cuda_exec = axom::CUDA_EXEC<CUDA_BLOCK_SIZE>;
+    retval = runMIR_cuda(mesh, options, resultMesh);
+  }
+  #endif
+  #if defined(AXOM_USE_HIP)
+  else if(policy == RuntimePolicy::hip)
+  {
+    retval = runMIR_hip(mesh, options, resultMesh);
+  }
+  #endif
+#endif
+  else
+  {
+    retval = -1;
+    SLIC_ERROR("Unhandled policy.");
+  }
+  timer.stop();
+  SLIC_INFO("Material interface reconstruction time: "
+            << timer.elapsedTimeInMilliSec() << " ms.");
+
+  // Output results
+  {
+    AXOM_ANNOTATE_SCOPE("save_output");
+    conduit::relay::io::blueprint::save_mesh(resultMesh, outputFilePath, "hdf5");
+  }
+
+  return retval;
+}
+
+//--------------------------------------------------------------------------------
+int main(int argc, char **argv)
+{
+  axom::slic::SimpleLogger logger;  // create & initialize test logger
+  axom::slic::setLoggingMsgLevel(axom::slic::message::Info);
```

Comment:
Minor: You can pass the level into the constructor
```suggestion
  axom::slic::SimpleLogger logger(axom::slic::message::Info);
```

Reply:
Changed it.